interop-tests/src/test/scala/akka/grpc/scaladsl/RichErrorModelNativeSpec.scala

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2020-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.scaladsl
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.grpc.internal.RichGrpcMetadataImpl
```

Comment:
So the user will need to directly access an Akka gRPC internal class match against?

Reply:
Yes. If this is the way to go, we should probably make this class non-internal to avoid confusion.

Reply:
I agree, that's not nice, we should make it public API.

Reply:
Added in [d052ee7](https://github.com/akka/akka-grpc/pull/1665/commits/d052ee7c1fbca2d2aa32fc007b0a4d3dcb95aaaa)